mayastor/src/core/bdev.rs

```diff
@@ -51,6 +50,22 @@ pub struct BdevStats {
     pub bytes_written: u64,
 }
 
+impl From<iscsi::Error> for CoreError {
```

Comment:
should these be done using snafu .context instead?

Reply:
Decided to do things "properly" and use .context() everywhere for the sake of consistency.